stl/inc/yvals_core.h

```diff
@@ -42,6 +42,7 @@
 //     (partially implemented)
 // P0898R3 Standard Library Concepts
 // P0919R3 Heterogeneous Lookup For Unordered Containers
+// P1227R2 Signed ssize() functions, unsigned size() functions

```

Comment:
We choose issue titles for the features that hopefully communicate the feature more precisely than do the WG21 proposal titles, and those issue titles go here. This one is "Signed `std::ssize()`, Unsigned `span::size()`" (#56).

Since you haven't implemented the `span` changes, this should have a `//     (partially implemented)` line immediately below as on line 42.